typescript/infra/config/environments/testnet4/core.ts

```diff
@@ -36,11 +38,26 @@ export const core: ChainMap<CoreConfig> = objMap(
         .map((origin) => [origin, defaultMultisigConfigs[origin]]),
     );
 
```

Comment:
is this the exact same logic as mainnet3? wonder if there's an easy opportunity for consolidation

Reply:
yup it is - it's almost identical except for the protocol fee. mainnet3 set to 0 wei, testnet4 is 1 wei